scripts/util/doc-builder.js

```diff
@@ -88,19 +88,17 @@ function checkoutSrcDirectories() {
   });
 }
 
-function copyHeadSrcDirectory() {
-  const version = 'head';
-  const srcRelativePath = path.join(options.srcDirRelPath, version);
-  const srcFullPath = path.join(process.cwd(), srcRelativePath);
-  const apiRelativePath = path.join(options.apiDirRelPath, version);
+function copySrcDirectory() {
```

Comment:
Yeah the only thing is you might want to generate docs before committing to head. At some point it would be good to write a watch and build docs command and this would mean you shouldn't need to commit to create docs.